LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 Greptime Team
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Apache License 2.0 text is expected to use AS IS.

See also https://github.com/apache/incubator-answer/pull/527#discussion_r1325686233.

If you read this APPENDIX closely, you will know that here is a template.